trustfall_core/src/graphql_query/error.rs

```diff
@@ -63,6 +63,9 @@ pub enum ParseError {
     #[error("Unrecognized transform operator: {0}")]
     UnsupportedTransformOperator(String, Pos),
 
+    #[error("Unrecognized fold directive: {0}")]
+    UnsupportedFoldDirective(String, Pos),
```

Comment:
Are you able to trigger this error case? (Or is it maybe already triggered by the existing test suite?)

If so, do you feel like this error message is sufficient to explain the problem and guide the user toward resolving it?

Reply:
Good callout 👍 I'll go ahead and take a look.